components/tikv_util/src/logger/file_log.rs

```diff
@@ -67,7 +67,11 @@ impl RotatingFileLogger {
 
         // Note: renaming files while they're open only works on Linux and macOS.
         let new_path = rotation_file_path_with_timestamp(&self.file_path, &Utc::now());
-        fs::rename(&self.file_path, new_path)?;
+        match fs::rename(&self.file_path, new_path) {
```

Comment:
it is better to use a failpoint to add a test here.

Reply:
Sounds like a good idea. Let me put one here.

Reply:
@siddontang failpoint 'file_log_rename' added.